test/runtests.jl

```diff
@@ -72,3 +72,9 @@ end
     @test Extents.intersection(nothing, nothing) === nothing
     @test Extents.intersection(nothing, b) === nothing
 end
+
+@testset "buffer" begin
+    a = Extent(X=(0.1, 0.5), Y=(1.0, 2.0))
```

Comment:
We need to test this with :

- arbitrary names
- some names missing (dont get padded)
- datetime as the pad

Reply:
done!

Reply:
Thanks, but I don't see any DateTime in there ;)